pysat/instruments/timed_saber.py

```diff
@@ -8,44 +8,42 @@
 
```

Comment:
Is this the best place for the "rules of the road"?

Reply:
How about as a note?

Reply:
Sure, or maybe reference (not positive that's right either)?  Is this info included in the ackn or ref functions?

Reply:
ackn and ref are still mostly unused throughout the instruments.  Been working on those as part of the 3.0 branch.